### PR TITLE
Add custom template support

### PR DESCRIPTION
**Pull request by martin-steinegger**

### CODE REVIEW

File: batch/AlphaFold2_batch.ipynb [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/batch/AlphaFold2_batch.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/martin-steinegger/ColabFold/blob/main/batch/AlphaFold2_batch.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (milot-mirdita) [inline batch/AlphaFold2_batch.ipynb:29]: This shouldn't change

File: colabfold/batch.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import os
 
+from Bio.PDB import MMCIFParser
```
Comment (milot-mirdita) [inline colabfold/batch.py:3]: Do we have an explicit dependency in poetry on biopython?

File: AlphaFold2.ipynb [hunk 3]
```diff
@@ -95,18 +93,23 @@
         "# number of models to use\n",
         "use_amber = False #@param {type:\"boolean\"}\n",
         "use_templates = False #@param {type:\"boolean\"}\n",
-        "save_to_google_drive = False #@param {type:\"boolean\"}\n",
-        "#@markdown -  if the save_to_google_drive option was selected, the result zip will be uploaded to your Google Drive\n",
-        "\n",
-        "#@markdown ### Advanced settings\n",
+        "upload_custom_templates = False #@param {type:\"boolean\"}\n",
```
Comment (milot-mirdita) [inline AlphaFold2.ipynb:96]: `use_templates`, `upload_custom_templates` are confusing labels, its not immediatly clear what the difference is and if you need to have use_templates enabled to use custom templates.

File: colabfold/batch.py (hunk 2 above)
Comment (martin-steinegger) [reply colabfold/batch.py:3]: yes.

File: batch/AlphaFold2_batch.ipynb (hunk 1 above)
Comment (martin-steinegger) [reply batch/AlphaFold2_batch.ipynb:29]: Very confusing. My version in martin-steinegger/ColabFold does not contain this line. 
```
$ grep martin-steinegger AlphaFold2_batch.ipynb
$
```
I will change it once merged.

File: AlphaFold2.ipynb (hunk 3 above)
Comment (martin-steinegger) [reply AlphaFold2.ipynb:96]: I agree. Maybe we could change it to "use_pdb70_template", "use_custom_template"? Would this be easier?

Comment (martin-steinegger) [reply AlphaFold2.ipynb]: I reworked the interface. Its now a drop down box. I also added explanation what the different options mean.

**Pull request merged**